sys/arm/arm/exception.S

```diff
@@ -48,6 +48,8 @@
 
 #include "assym.inc"
 
+#include <sys/intr.h>
```

Comment:
The only place which should `#include <sys/intr.h>` are the `machine/intr.h` headers.  Everywhere else should pick up `sys/intr.h` by `#include <machine/intr.h>`.